src/actions/actionCreators.ts

```diff
@@ -0,0 +1,11 @@
+import { ADD_TODO } from './actions';
+
+// export const addTodo = (description, timestamp) => {
+//     return {
+//         type: ADD_TODO,
+//     payload: {
+//         description,
```

Comment:
1) do not push commented code
2) configure your project to use the same style (single quotes or double quotes) coz somewhere I saw double quotes